README.md

```diff
@@ -70,52 +64,37 @@ IOmfIngressService omfIngressService = baseOmfIngressService.GetOmfIngressServic
 
-## OMF Connections
-
-An OMF Connection is made up of three components: one or more Clients, a Topic, and a Subscription. Data is sent to a Topic via a Client, where the data is buffered and made available for the Subscription. The Subscription relays data from the Topic to the Sequential Data Store in the namespace that the Subscription resides in. The OmfIngressClient in this example creates a connection using an existing clientId, and creating a Topic and Subscription as detailed below.
-
 ## Clients
 
-Devices sending OMF messages each need a clientId and clientSecret. The clientId is used route messages to the proper topic(s). ClientIds may be mapped to at most one topic per namespace. For more details on Clients see the [Client Credential Client Documentation](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.html).
+Devices sending OMF messages each need a clientId and clientSecret. The clientId is used route messages to the proper connection(s). ClientIds may be mapped to at most one connection per namespace. For more details on Clients see the [Client Credential Client Documentation](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.html).
 
-## Topics
+## OMF Connections
 
-A Topic is used to aggregate data received from clients and make it available for consumption via a Subscription. A topic must contain at least one client Id. Client Ids may be added to or removed from an existing topic. First, we create the Topic locally by instantiating a new Topic object:
+ An OMF Connection is the connection that handles reading and handling OMF messages. We create the Connection locally by instantiating a new OmfConnectionCreate object:
 
 ```C#
-Topic topic = new Topic()
+OmfConnectionCreate omfConnectionCreate = new OmfConnectionCreate()
 {
-    Name = "REPLACE_WITH_TOPIC_NAME",
-    Description = "This is a sample Topic",
+    Name = "REPLACE_WITH_CONNECTION_NAME",
+    Description = "This is a sample Connection",
     ClientIds = new List<string>() { mappedClientId }
 };
 ```
 
-Then use the Ingress client to create the Topic in ADH:
+Then use the Ingress client to start the creation of the Client:
 
 ```C#
-Topic createdTopic = await omfIngressService.CreateTopicAsync(topic);
+OmfConnection omfConnection = await omfIngressService.BeginCreateOmfConnectionAsync(omfConnectionCreate).ConfigureAwait(false);
 ```
 
-## Subscriptions
-
-A Subscription is used to consume data from a Topic and relay it to the Sequential Data Store. First, we create the Subscription locally by instantiating a new Subscription object:
+After initiating creation, we can monitor the status of the omfConnection by continuously querying for the connection and checking its status:
```

Comment:
i know we are doing it continuously but I think we could drop continuously here unless we want to make a point that we are waiting for the connection to be valid before continuing, but I think of this documentation as being like API docs where we should be highlighting the API and then in the sample comments we can talk about why we are doing something (or in the steps of the sample)

Reply:
eh, re-reading this I am thinking that we should highlight while we are in this loop

Reply:
Changed the phrasing. Let me know if it reads better.